emu/src/cpu/data_processing.rs

```diff
@@ -1,19 +1,15 @@
+use crate::bitwise::Bits;
```

Comment:
I think this file contains only the Dataprocessing instruction for Arm mode. Should we move this login into the new file `arm/alu_instructions.rs`?

Reply:
For me it's ok apart from this. But if you think we should tackle this in another PR I'm fine with it @guerinoni